### DISCUSSION TITLE
automatic docs generation places Shopware docs link into all recipe docs

### DISCUSSION THREAD

**Discussion by M-arcus** [deployphp/deployer#3918] [Bugs] [answered]
### Deployer Version

v7.4.1

### Target OS

Ubuntu 22.04

### Which PHP version are you using?

PHP 8.3

### Steps to reproduce

As mentioned in https://github.com/deployphp/deployer/issues/3831 by @SimJoSt 
> Since https://github.com/deployphp/deployer/commit/d34b80f3f2b11d438a6fec3a9f5d7151d7190670 through https://github.com/deployphp/deployer/pull/3816 the automatic docs generation commit change the following line from
> 
>   * [deploy:update_code](/docs/recipe/deploy/update_code.md#deployupdate_code) – Updates code
> to
> 
>   * [deploy:update_code](/docs/recipe/shopware.md#deployupdate_code) – 
> for all recipes, not just the Shopware one.
> 
> Here are some of the commits of the automatic docs generation that I observed this in:
> 
> https://github.com/deployphp/deployer/commit/04c592686088f3437fa5d465e78a9c79109eff8d
> https://github.com/deployphp/deployer/commit/c859acec70718bb7c4590cddebd02cabbb976e28
> https://github.com/deployphp/deployer/commit/04c592686088f3437fa5d465e78a9c79109eff8d
> I'm not sure why this is happening, as it does not occur when I run bin/docsgen on my local machine with the same commit.

**Comment by antonmedv** [comment, marked as answer]
Yeap. It was a bug in docgen. I fixed it.
